openedx/core/djangoapps/organization_api/tests/test_views.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for organization API.
+"""
+import json
+import unittest
```

Comment:
json and unittest are part of the standard library and belong in their own block of imports.